Pinta.Core/Managers/LivePreviewManager.cs

```diff
@@ -303,25 +292,10 @@ protected override void OnCompletion (
 		}
 	}
 
-	void FireLivePreviewEndedEvent (RenderStatus status, Exception? ex)
-	{
-		if (Ended == null) return;
-		LivePreviewEndedEventArgs args = new (status, ex);
-		Ended (this, args);
-	}
-
 	void FireLivePreviewRenderUpdatedEvent (double progress, RectangleI bounds)
```

Comment:
`FireLivePreviewRenderUpdatedEvent()` should also be renamed now that it's not firing any events :)
Perhaps just `HandleUpdate()` would work, to go along with the existing `HandleCancel()` and `HandleApply()`?

Reply:
@cameronwhite  I have renamed it :)